django_redshift_backend/base.py

```diff
@@ -87,7 +87,8 @@ def get_db_converters(self, expression):
             converters.append(self.convert_uuidfield_value)
         return converters
 
-    def convert_uuidfield_value(self, value, expression, connection, context):
+    def convert_uuidfield_value(self, value, expression, connection,
+                                context=None):
```

Comment:
Please remove `context` argument completely because `context` has been removed since django 2.0 and django-redshift-backend already drop support django 1.x.

Reply:
context removed !